js/id/modes/add_area.js

```diff
@@ -11,13 +11,20 @@ iD.modes.AddArea = function(context) {
             .tail(t('modes.add_area.tail'))
             .on('start', start)
             .on('startFromWay', startFromWay)
-            .on('startFromNode', startFromNode),
-        defaultTags = {area: 'yes'};
+            .on('startFromNode', startFromNode);
+
+    function defaultTags() {
+        var tags = {area: 'yes'};
+        if (context.indoor().enabled()) {
+            tags.level = context.indoor().level();
+        }
+        return tags;
+    }
```

Comment:
I dont particulary like this code, b/c it repeats in all add_* files. Still i dont think it should use other dependency.. So maybe its fine.